### ISSUE TITLE
Memory Pak save restore

### ISSUE THREAD

**Issue by Crowzilla** [ottelo9/altra64#1]
How i can different Backups? i cabt find the option for that

**Comment by ottelo9** [comment]
What do you mean?

**Comment by Crowzilla** [comment]
I am using a extern mempak tool to save my memory pak , because when i use the backup function of Altra64 is doesnt backup it or if it does i cant see the restore function

**Comment by ottelo9** [comment]
Yes, I also use an external tool for this, because this function is broken. But try to use https://github.com/Suprapote/Altra64. He tries to fix it.
